yt_dlp/extractor/rumble.py

```diff
@@ -236,6 +222,96 @@ def _real_extract(self, url):
         }
 
 
+class RumbleIE(InfoExtractor):
+    _VALID_URL = r'''(?x)
+        https?://(?:www\.)?rumble\.com/
+        (?P<id>v            # ids start with v
+            (?!ideos)       # exclude https://rumble.com/videos
+            [0-9a-zA-Z]{4,} # video page identifier (case insensitive)
+            (?:-[^&?#$/]*)? # usually followed by -<title>.html otherwise redirected
+        )
+        [^/]*$              # queries are allowed, but no slashes
+    '''
+    _EMBED_REGEX = [r'<a class=video-item--a href=(?P<url>/v[0-9a-z.-]+\.html)>']
+    _TESTS = [{
+        'add_ie': ['RumbleEmbed'],
+        'url': 'https://rumble.com/vdmum1-moose-the-dog-helps-girls-dig-a-snow-fort.html',
+        'md5': '53af34098a7f92c4e51cf0bd1c33f009',
+        'info_dict': {
+            'id': 'vb0ofn',
+            'ext': 'mp4',
+            'timestamp': 1612662578,
+            'uploader': 'LovingMontana',
+            'channel': 'LovingMontana',
+            'upload_date': '20210207',
+            'title': 'Winter-loving dog helps girls dig a snow fort ',
+            'description': 'Moose the dog is more than happy to help '
+                           'with digging out this epic snow fort. Great job, Moose!',
```

Comment:
I have noticed u do this in many PRs https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#long-lines-policy
```suggestion
            'description': 'Moose the dog is more than happy to help with digging out this epic snow fort. Great job, Moose!',
```
or even better, use md5

Reply:
I accept a projects code conventions, but I struggle sometimes without tool support...